src/pybel/__init__.py

```diff
@@ -45,6 +45,7 @@
 from .io.umbrella_nodelink import to_umbrella_nodelink, to_umbrella_nodelink_file, to_umbrella_nodelink_gz
 from .manager import Manager, from_database, to_database
 from .parser.parse_bel import parse
+from .schema import is_valid_edge, is_valid_node
```

Comment:
I'd rather this weren't exposed at the top level. Anyone who wants to get into the nitty-gritty of the schema should be comfortable importing it with `from pybel.schema import is_valid_edge, is_valid_node`